content/articles/why-you-should-use-wakanda/index.md

```diff
@@ -0,0 +1,55 @@
+
+### Features of Wakanda JavaScript Framework 
+There are very nice features which make Wakanda JavaScript stand out in this modern day. 
+
+#### Wakanda database
+The Wakanda Database itself stands out for me, especially when it is paired perfectly with the Wakanda Server. It offers the same experience as MongoDb and CouchDb since it uses modern NoSQL with similar portability as SQLite. Wakanda JavaScript framework saves us stress and time since it is portable and ready to use. 
+
+#### Wakanda server
+The Wakanda server runs and hosts the backend applications in the course of production and development. It is a multi-threaded HTTP server which incorporates server-side JavaScript APIs unmasked in REST. Everything needed is already written for the users in JavaScript. It also saves time since the user does not have to go into configuration tasks. The Wakanda server is also portable. It lowers the cost of both configuration and development by using pre-bundled components.
+
+##### JavaScript application server
+The Wakanda JavaScript app server uses the V8 engine which operates on JavaScript runtime. It exhibits a rich synchronous API to hold filesystems, sockets remote databases and workers. It support asynchronous development style.
+
+##### HTTP server
+The Wakanda HTTP server exposes the autogenerated Wakanda REST API. It is also responsible for serving static application files.
+
+##### Data abstraction layer
+The data abstraction layer is essential for data security. It helps in data manipulation through use of a JavaScript API. It exposes an automated REST API to manipulate and access data.
+
+##### Authentication (directory)
+It helps in exposing both the REST API and server side JavaScript API. It is responsible for authentication of management sessions and users.
+
+#### Wakanda studio
+The Wakanda studio helps to develop and test apps locally together with troubleshooting your environment. In this era where everyone is craving for a code editor, Wakanda has taken a different turn in coming up with an IDE. 
+
+Wakanda's main objective is to give a quick and easy way in project scaffolding. Wakanda is the solution. The philosophy behind Wakanda studio which simplifies developers lives who have decided to work with Wakanda JavaScript framework stack is called RAD (Rapid Application Development).
+
+Do you love React? You can change the scaffolding of your project to adapt to your desire. Wakanda identifies and acknowledges the three critical parts in all applications.
+
+These crucial parts are all used in the initial scaffolding. They include the backend, web and the mobile. Studio has a code editor emanating from extension updatable, an application prototype, deploy solutions, code mirror and git integration. This studio gives the user the whole view of application.
```

Comment:
Be specific. Web is very huge. Mobile is very huge too. Use specified terms such as "Android, or Web applications".

The word "Studio" can mean anything in this context. Android Studio, Visual Studio, etc. Use the full names.

Convert what the "studio" emanates into a list.